frontend/src/common-components/layout/CmschLayout.tsx

```diff
@@ -3,26 +3,32 @@ import * as React from 'react'
 import { Helmet } from 'react-helmet'
 import { Footer } from '../footer/Footer'
 import { HasChildren } from '../../util/react-types.util'
-import { APP_NAME } from '../../util/configs/environment.config'
 import { Navbar } from '../navigation/Navbar'
 import { Warning } from '../Warning'
 import { ScrollToTop } from './ScrollToTop'
+import { useConfigContext } from '../../api/contexts/config/ConfigContext'
 
 type Props = {
   background?: string
 } & HasChildren
 
-export const CmschLayout = ({ background, children }: Props) => (
-  <>
-    <Helmet titleTemplate={`%s | ${APP_NAME}`} defaultTitle={APP_NAME} />
-    <Flex direction="column" minHeight="100vh">
-      <ScrollToTop />
-      <Navbar />
-      <Box background={background} flex={1} pb={20}>
-        <Warning />
-        {children}
-      </Box>
-      <Footer />
-    </Flex>
-  </>
-)
+export const CmschLayout = ({ background, children }: Props) => {
+  const config = useConfigContext()
+  return (
+    <>
+      <Helmet
+        titleTemplate={`%s | ${config?.components.app.siteName || 'CMSch'}`}
+        defaultTitle={config?.components.app.siteName || 'CMSch'}
```

Comment:
Helmet does not like this, because it could change on render-time, not on first content-painting time. We should switch to `react-helmet-async`, or disregard SEO at once xd